livre/lib/pipeline/HistogramFilter.cpp

```diff
@@ -80,6 +80,7 @@ struct HistogramFilter::Impl
         const auto& frustums = input.get< Frustum >( "Frustum" );
         const auto& viewports = input.get< Viewport >( "RelativeViewport" );
 
+        auto dataSourceRange = input.get< Vector2f >( "DataSourceRange" ).front();
         const auto& frustum = frustums.front();
```

Comment:
Why don't you do the same thing for frustum and viewport, I mean: 

    const auto& frustum = input.get< Frustum >( "Frustum" ).front();